engines/dfc_provider/app/controllers/dfc_provider/enterprise_groups_controller.rb

```diff
@@ -3,6 +3,16 @@
 # OFN EnterpriseGroup is represented as DFC Enterprise.
 module DfcProvider
   class EnterpriseGroupsController < DfcProvider::ApplicationController
+    def index
+      person = PersonBuilder.person(current_user)
```

Comment:
Why is there a Person?

Reply:
> Why is there a Person?

The DFC always requires a person. It operates on a graph with the current user as the root from which all data is connected. Without the person, the consuming app wouldn't know how to store the information. Well, that's true for pure Semantic Web apps. We will probably write apps that consume the data anyway and ignore the person, maybe.